.travis.yml

```diff
@@ -170,26 +177,30 @@ jobs:
     if: type IN (pull_request)
     env:
     - NODE_ENV=production
-    - BUNDLE_ANALYSIS=1
     - PORT=12345
-    - VERSION=$TRAVIS_PULL_REQUEST_BRANCH
-    - SENTRY_ENVIRONMENT=pr
+    - VERSION=$TRAVIS_BRANCH
+    - SENTRY_ENVIRONMENT=staging
+    - BUNDLE_ANALYSIS=1
     - STREAMR_API_URL=https://staging.streamr.com/api/v1
     - STREAMR_WS_URL=wss://staging.streamr.com/api/v1/ws
     - PLATFORM_ORIGIN_URL=https://staging.streamr.com
     - STREAMR_URL=https://staging.streamr.com
+    - PLATFORM_PUBLIC_PATH=https://marketplace-staging.streamr.com
```

Comment:
I think this is OK, let's double check it all still works after merge. I can't remember the exact magic that makes these kind of builds happy 🙈